INSTALL.pipeline

```diff
@@ -45,6 +45,23 @@ him know a username you'd like for yourself, if you don't already have one.
 He will set things up so your new pipeline server can coordinate with the 
 others, and will be allowed to upload finished WARCs to the Internet Archive.
 
+** STEP 2.5: PYTHON VENV **
+
+For a recent OS, you will likely want to run Archivebot in a venv.  These
+instructions use the installer; we suggest readng the pyenv.run script first
+or installing it manually.
+
+    sudo apt install git make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev libffi-dev curl
+    curl https://pyenv.run | bash
+    echo 'export PATH="$HOME/.pyenv/bin:$PATH"' >> ~/.bashrc
+    echo 'export "$(pyenv init -)"' >> ~/.bashrc
+    echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
```

Comment:
This isn't quite right. Things changed a bit in pyenv in May (https://github.com/pyenv/pyenv/pull/1898). The background is that `PATH` modifications belong in `~/.profile`, not `~/.bashrc`, because the latter may be sourced multiple times. `pyenv init` does `PATH` manipulations as well as other things that need to be in `~/.bashrc` (namely shell integration via a function). But on some systems, `~/.profile` sources `~/.bashrc` when invoked from a Bash shell, which makes things even messier...

Here are all the details: https://github.com/pyenv/pyenv#basic-github-checkout

I've never used the installer/pyenv.run, but I think it prints instructions after installation on how to correctly configure things? Can we just refer to those instead of listing it explicitly here?